ext/chart_kit/info.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<extension key="chart_kit" type="module">
+  <file>chart_kit</file>
+  <name>Chart Kit</name>
+  <description>This extension provides additional Search Display types for Search Kit for making charts, using the dc/d3 library.</description>
+  <license>AGPL-3.0</license>
+  <maintainer>
+    <author>Benjamin Walpole</author>
+    <email>ben@civicrm.org</email>
+  </maintainer>
+  <urls>
+    <url desc="Documentation">https://docs.civicrm.org/user/en/latest/the-user-interface/search-kit/</url>
+    <url desc="Chat">https://chat.civicrm.org/civicrm/channels/search-improvements</url>
```

Comment:
wondering about whether to create another MM channel separate from the main Search Kit one.

@colemanw do you have a view?

Reply:
I made https://chat.civicrm.org/civicrm/channels/dataviz